### ISSUE TITLE
no such directory ./templates

### ISSUE THREAD

**Issue by stefjoosten** [AmpersandTarski/Ampersand#1276]
# What happened
I tried to compile a prototype using just the Ampersand compiler
```
sjo00577@BA92-VYF9TXMD9G % docker run -it --platform=linux/amd64 -v "$(pwd)":/scripts ampersandtarski/ampersand:v4.6.2 check foo.adl
This script of Foo contains no type errors.
sjo00577@BA92-VYF9TXMD9G % docker run -it --platform=linux/amd64 -v "$(pwd)":/scripts ampersandtarski/ampersand:v4.6.2 proto foo.adl 
Generating frontend...
Error while reading template.
File could not be read: .proto/templates/Atomic-OBJECT.html
.proto/templates/Atomic-OBJECT.html: openFile: does not exist (No such file or directory)
ExitFailure 130
sjo00577@BA92-VYF9TXMD9G % 
```
# What I expected
I expected this file to generate some code, because foo.adl contains no errors

# Version of ampersand that was used
Ampersand-v4.6.2 [51c3de2e65c540ef026925fe8547c991765a5588:refs/tags/v4.6.2]

**Comment by Michiel-s** [comment]
Indeed, as @hanjoosten explained correctly templates files are decoupled. That is because they need to be always in sync with the current framework version to work.

@stefjoosten You can run the proto command with the `--no-frontend` command. This will skip the frontend files that need the templates, but does generate backend configuration files, like the sql queries and more.

**Comment by stefjoosten** [comment]
Aha, we have a workaround!

**Comment by stefjoosten** [comment]
that sounds like plenty of ideas. I suggest we two things:
* Let us make the option `--no-frontend` a default option for `ampersand proto`, to make the decoupling perfect. An unknowing user (like me... :-) ) should not stumble into this error. Consequently, we have to add the option`--frontend` to the Dockerfiles that generate prototypes (or add the correct environment variable for the compiler if there is one).
* Let us improve the error message with meaningful text and a hint of what to do.

Agreed?
